### DISCUSSION TITLE
Get more info after adding domains using api

### DISCUSSION THREAD

**Discussion by advany** [vercel/vercel#8308] [Help] [answered]
Hi,

After I add a domain using the api. Where do I get the information on what api what to do as a challenge (IP to set dns for, cname, nameservers or even txt when domain is owned by another team). How can I get this info using the api?

Regards,

Ab

**Comment by amyegan** [comment, marked as answer]
The API does not currently provide the information for A records or nameservers that will need to be set when [adding a custom domain to a project](https://vercel.com/docs/rest-api#endpoints/projects/add-a-domain-to-a-project). If you add a domain to your Project via the API, you can visit your project domain settings online to find the nameservers, A, CNAME, or TXT records that need to be set.

You also have the option to [add a domain via Vercel CLI](https://vercel.com/docs/cli#commands/domains), where the DNS information you need will be printed to the terminal. 
```
~/code/vercel-test main*
❯ vercel domains add mynewdomain.dev
Vercel CLI 27.2.0
⠹ Adding domain mynewdomain.dev to project vercel-test> Success! Domain mynewdomain.dev added to project vercel-test. [472ms]
WARN! This domain is not configured properly. To configure it you should either:
  a) Set the following record on your DNS provider to continue: `A mynewdomain.dev 76.76.21.21` [recommended]
  b) Change your Domains's nameservers to the intended set
     Intended Nameservers    Current Nameservers
     ns1.vercel-dns.com      -                      ✘
     ns2.vercel-dns.com      -                      ✘

  We will run a verification for you and you will receive an email upon completion.
  Read more: https://vercel.link/domain-configuration
```

**Reply by advany** [reply]
@amyegan is the A record always the IP 76.76.21.21 and the nameservers are they always ns1.vercel-dns.com and ns2.vercel-dns.com?

**Reply by amyegan** [reply]
I can't say that they'll never ever change, but that's what they always are for the foreseeable future. The only one that would be different is cases where you need a TXT because the domain is already used by another account.
